src/Method/Definition.php

```diff
@@ -172,92 +138,47 @@ public function setPrototypes(array $prototypes)
     /**
      * Get all prototypes
      *
-     * @return array $prototypes Array of \Laminas\Server\Method\Prototype objects or arrays
+     * @return Prototype[] $prototypes
      */
-    public function getPrototypes()
+    public function getPrototypes(): array
     {
         return $this->prototypes;
     }
 
-    /**
-     * Set method help
-     *
-     * @param  string $methodHelp
-     * @return \Laminas\Server\Method\Definition
-     */
-    public function setMethodHelp($methodHelp)
+    public function setMethodHelp(string $methodHelp): self
     {
-        $this->methodHelp = (string) $methodHelp;
+        $this->methodHelp = $methodHelp;
         return $this;
     }
 
-    /**
-     * Get method help
-     *
-     * @return string
-     */
-    public function getMethodHelp()
+    public function getMethodHelp(): string
     {
         return $this->methodHelp;
     }
 
-    /**
-     * Set object to use with method calls
-     *
-     * @param  object $object
-     * @throws Server\Exception\InvalidArgumentException
-     * @return \Laminas\Server\Method\Definition
-     */
-    public function setObject($object)
+    public function setObject(object $object): self
     {
-        if (! is_object($object) && (null !== $object)) {
-            throw new Server\Exception\InvalidArgumentException(sprintf(
-                'Invalid object passed to %s',
-                __METHOD__
-            ));
-        }
         $this->object = $object;
         return $this;
     }
 
-    /**
-     * Get object to use with method calls
-     *
-     * @return null|object
-     */
-    public function getObject()
+    public function getObject(): ?object
```

Comment:
if null is allowed to be returned, the `setObject()` need to allow null parameter.

Reply:
There is already a discussion about that on the setter...